clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -80,7 +81,7 @@ trait GvcfTests { self: BaseIntegrationSpec =>
       sampleAlias = s"someAlias $randomId",
       version = 2,
       documentStatus = Some(DocumentStatus.Normal),
-      gvcfPath = Some(URI.create("gs://path/gvcf.gvcf"))
+      gvcfPath = Some(URI.create("gs://path/gvcf.g.vcf.gz"))
```

Comment:
If we have to hardcode these values in the transfer metadata, it would be nice to refer to it here instead of duplicating the hardcoded values.